### PR TITLE
Adds radial mode selection menu into Nanosuit [READY]

### PR DESCRIPTION
**Pull request by yoyobatty**
[Guidelines]: # (Be sure that your PR follows our guidelines, such as modularization and comment standards. You can read more about the subject here: https://github.com/HippieStation/HippieStation/blob/master/hippiestation/README.md )

[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs )

:cl: YoYoBatty
add: The nanosuit now has a radial mode selection menu which you can activate by holding C.

Suicide implant has better logging and actions have been condensed.
Also upgraded nanosuit helper description to say damage is blocked at 50%
/:cl:

[why]: Because it's cool as fuck.

### CODE REVIEW

**Comment by JohnGinnane** [comment]
very cool

**Comment by ghost** [comment]
It'd be convenient to be able to have it set to middle-click

**Comment by ghost** [comment]
middle mouse button swaps hands so no

**Comment by ghost** [comment]
it does?

**Comment by Eagle-Eyess** [comment]
+1

**Comment by ExpAntique** [comment]
Thank you yoyo, very cool

 +1

**Comment by HippieBartels** [comment]
Fancy stuff +1

File: hippiestation/code/modules/clothing/spacesuits/nanosuit.dm [hunk 1]
```diff
@@ -1065,3 +1059,73 @@
 /obj/item/stock_parts/cell/nano
 	name = "nanosuit self charging battery"
 	maxcharge = 100
+
+mob/living/carbon/human/key_down(_key, client/user)
```
Comment (Carbonhell) [inline hippiestation/code/modules/clothing/spacesuits/nanosuit.dm:1065]: What the hell no

**Review by Carbonhell — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
+1

File: hippiestation/code/modules/clothing/spacesuits/nanosuit.dm [hunk 2]
```diff
@@ -1065,3 +1061,73 @@
...
```
Comment (Carbonhell) [inline hippiestation/code/modules/clothing/spacesuits/nanosuit.dm:1065]: are you sure key_down/up don't require any return value?

**Review by Carbonhell — APPROVED** [review]

**Comment by Carbonhell** [comment]
Other redefinitions return ..() but it seems useless. merging

**Pull request merged**